felt/caigo.go

```diff
@@ -1,4 +1,4 @@
-package caigo
+package felt
```

Comment:
Prefer for this to move to a `pedersen` or `curve` package and for the functions to be exposed as plan functions rather than struct members. So a caller can do: `pedersen.Verify(...)`, `pedersen.Sign(...)` rather than creating a curve instance each time

Reply:
You are right! But we might need several curve too